src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -134,6 +134,7 @@ $pf-global--spacer--form-element: pf-size-prem(6px) !default;
 
 // Gutter
 $pf-global--gutter: $pf-global--spacer--lg !default;
+$pf-global--gutter--md: $pf-global--spacer--md !default;
```

Comment:
Are you naming this variable by the breakpoint that it relates to? IMO when i first glance at the variable it seems like its a larger version of the default variable which could be confusing for users since it really is smaller. I wonder if it should be named with sm because its a smaller version of the gutter, in the case that the variable is used elsewhere when it doesnt relate to a breakpoint? I think most of the variables in here are named based on size over breakpoint, but let me know your reasoning :)

Reply:
Not by the breakpoint - I'm using `--md` as the modifier to represent "medium" since that's the size spacer it maps to. I wondered about this, too. The way I see it, gutters are similar to our spacing system, but they are used in a different context. We could potentially have gutters that map 1:1 with our spacers. So I think having gutter names that match the spacer names makes sense. In this case, I'm going off the assumption that `pf-global--gutter` is the same as `pf-global--gutter--lg`, so `pf-global--gutter--md` is `pf-global--spacer--md`, `pf-global--gutter--sm` would map to `pf-global--spacer--sm`, etc. If they don't match, I think that's confusing, and if we decided to have a gutter that matched `pf-global--spacer--xs`, we would need to create `pf-global--gutter--xxs`.

Reply:
Ok great that makes sense and I like that approach